why.js

```diff
@@ -205,6 +206,54 @@ module.exports = function whyNotEqual(value, other) {
 		if (isProto.call(other, value)) { return 'second argument is the [[Prototype]] of the first'; }
 		if (getPrototypeOf(value) !== getPrototypeOf(other)) { return 'arguments have a different [[Prototype]]'; }
 
+		var valStringPrimitive = null;
+		var valStringPrimitiveErr = null;
+		try {
+			valStringPrimitive = toPrimitive(value, String);
+		} catch (error) {
+			valStringPrimitiveErr = error;
+		}
+
+		var otherStringPrimitive = null;
+		var otherStringPrimitiveErr = null;
+		try {
+			otherStringPrimitive = toPrimitive(other, String);
+		} catch (error) {
+			otherStringPrimitiveErr = error;
+		}
+
+		if (valStringPrimitiveErr || otherStringPrimitiveErr) {
+			if (!valStringPrimitiveErr) { return 'second argument toString throws; first does not'; }
+			if (!otherStringPrimitiveErr) { return 'first argument toString throws; second does not'; }
```

Comment:
because of Symbol.toPrimitive, i don't think we can reliably say `valueOf` or `toString` in these explanations

Reply:
Got it, will update to"to primitive" tomorrow.